integrationtests/cli/helm/helm_test.go

```diff
@@ -0,0 +1,160 @@
+package helm
+
+import (
+	"io/fs"
+	"path/filepath"
+
+	"github.com/rancher/fleet/integrationtests/cli"
+	"github.com/rancher/fleet/modules/cli/apply"
+	"github.com/rancher/fleet/pkg/bundlereader"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	numberOfFilesInHelmTestRepo = 9
+)
+
+var repo = repository{}
+
+var _ = Describe("Fleet apply helm release", Ordered, func() {
+	When("apply a folder with fleet.yaml that contains a helm release", func() {
+		When("no auth is required", func() {
+			BeforeAll(func() {
+				repo.startRepository(false)
```

Comment:
I'm not sure why this works 🤷 
Tests are executed in parallel and repo always uses the same port? Shouldn't this run into a port conflict.

Reply:
We are using [Ordered](https://onsi.github.io/ginkgo/#ordered-containers), which will not run tests in parallel to avoid this problem